.github/workflows/ci-Instrumentation.Process.yml

```diff
@@ -0,0 +1,41 @@
+name: OpenTelemetry.Instrumentation.Process
+
+on:
+  pull_request:
+    branches: [ 'main*' ]
+    paths:
+    - 'src/OpenTelemetry.Instrumentation.Process/**'
+    - 'test/OpenTelemetry.Instrumentation.Process.Tests/**'
+
+jobs:
+  build-test:
+
+    strategy:
+      fail-fast: false # ensures the entire test matrix is run, even if one permutation fails
+      matrix:
+        os: [ windows-latest, ubuntu-latest ]
+        version: [ net462, netcoreapp3.1, net6.0, net7.0 ]
+        exclude:
+        - os: ubuntu-latest
+          version: net462
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install .NET 3.1 SDK
+      uses: actions/setup-dotnet@v3.0.3
+      with:
+        dotnet-version: '3.1.x'
+
+    - name: Install .NET 7 SDK
+      uses: actions/setup-dotnet@v3.0.3
+      with:
+        dotnet-version: '7.0.x'
+
+    - name: Build
```

Comment:
```suggestion
    - name: Build Instrumentation.Process
```

Reply:
Let's leave these naming discussion for the next PR.
Ideally I would hope:

"{component name} \ {name of the CI step, e.g. install runtime / restore package / build / test}"